.rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+unstable_features = true
+indent_style = "Block"
+use_field_init_shorthand = true
+reorder_imports = false
```

Comment:
I would keep the reordering, sometimes it decreases the size of the patches.

Reply:
Looks good to me too: +1